src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { UserService } from 'app/core/user/user.service';
 import { AccountService } from 'app/core/auth/account.service';
 import { MockAccountService } from '../../helpers/mocks/service/mock-account.service';
 import { MockActivatedRoute, MockRouter } from '../../helpers/mocks/service/mock-route.service';
```

Comment:
I think you missed the Codacy warning here :)

Reply:
I did, thanks for the hint. Fixed in https://github.com/ls1intum/Artemis/pull/2474/commits/7ba17b6c9528ef7291ce2cbd1d768938afda852c.